docs/index.md

```diff
@@ -65,29 +85,30 @@ demo-0.1.0.tar.gz
 $ pip install dist/demo-0.1.0.tar.gz
 ```
 
-... or create a **ready-to-go Flask web service**:
+After installing the package, another user can invoke the model via the CLI or by importing the `demo` module like we did earlier.
+
+#### Deploy a web service
+
+You can deploy the model inference web service to Heroku ([a free account](https://signup.heroku.com/) works fine with this demo):
 
 ```
-$ whisk app start
-$ curl --location --request POST 'http://localhost:5000/predict' \
---header 'Content-Type: application/json' \
---data-raw '{"data":[[0, 1], [2, 3]]}'
+$ whisk app create demo-[INSERT YOUR NAME]
 ```
 
-... and **deploy the web service to Heroku** ([a free account is fine](https://signup.heroku.com/)):
+### What next?
```

Comment:
What are your thoughts on encouraging someone to "Adjust the example model"? This would encourage someone to get a feel for the actual `src` code, and see instant results from their changes flowing through the whisk functionality. I think it was actually the first thing I tried when opening whisk. I was also talking to a DS friend about this, and they immediately asked "where's the model?". Maybe something like this:

#### Change the example model

To change the example model, head over to `src/demo/models/model.py` and make a small change to the `predict` method. After saving this, you'll see all whisk methods automatically update with this new logic.

Reply:
Good idea. We can do that. We can also generate the demo project and put it on GitHub and link to the source code.

Reply:
Nice yeah thats a great idea - I added a tasks for this and the other demo in #10